app/src/main/java/org/openobservatory/ooniprobe/activity/MainActivity.java

```diff
@@ -66,38 +76,32 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
                 }
             });
             bottomNavigation.setSelectedItemId(getIntent().getIntExtra(RES_ITEM, R.id.dashboard));
-            if (getPreferenceManager().getAppOpenCount() != 0
-                    && getPreferenceManager().getAppOpenCount() % PreferenceManager.AUTOTEST_DIALOG_COUNT == 0
-                    && !getPreferenceManager().isAutomaticTestEnabled()
-                    && !getPreferenceManager().isAskAutomaticTestDialogDisabled()) {
+            if (notificationManager.shouldShow()) {
```

Comment:
These are inverted, first ask for autorun, else notification.
see https://github.com/ooni/probe-android/blob/master/app/src/main/java/org/openobservatory/ooniprobe/activity/MainActivity.java#L69